pycid/export/gambit.py

```diff
@@ -0,0 +1,223 @@
+from __future__ import annotations
+
+import itertools
+from collections import defaultdict
+from functools import partial, update_wrapper
+from typing import Any, Callable, Dict, Hashable, Iterable, KeysView, List, Mapping, Optional, Set, Tuple, Union
+
+import pygambit
+
+from pycid.core.cpd import StochasticFunctionCPD
+from pycid.core.macid_base import MACIDBase
+
+Outcome = Any
+
+
+def macid_to_efg(
+    macid: MACIDBase,
```

Comment:
maybe add a comment after this first argument saying:
# expected input is a MACID object, but also allow for CIDs to be converted to EFGs